CMakeLists.txt

```diff
@@ -266,8 +266,9 @@ if(LOVR_USE_STEAM_AUDIO)
   elseif(WIN32)
     set_target_properties(Phonon PROPERTIES IMPORTED_IMPLIB "${LOVR_STEAM_AUDIO_PATH}/lib/Windows/x64/phonon.lib")
     set_target_properties(Phonon PROPERTIES IMPORTED_LOCATION "${LOVR_STEAM_AUDIO_PATH}/bin/Windows/x64/phonon.dll")
-  elseif(APPLE) # This should be possible, there is a dylib in OSX/
-    message(FATAL_ERROR "LOVR_USE_STEAM_AUDIO is not currently supported on Apple platforms")
+  elseif(APPLE)
+    set_target_properties(Phonon PROPERTIES IMPORTED_LOCATION "${LOVR_STEAM_AUDIO_PATH}/lib/OSX/libphonon.dylib"
+                                            BUILD_RPATH "${LOVR_STEAM_AUDIO_PATH}/lib/OSX")
```

Comment:
I would prefer to `move_lib` the library and fix the rpath on regular macOS builds to be `@executable_path` (basically move `MACOSX_RPATH`, `BUILD_WITH_INSTALL_RPATH`, and `INSTALL_RPATH` properties outside of `if (LOVR_BUILD_BUNDLE)`).

It is a little bit more involved because `move_lib` needs to work properly on non-bundled builds.  At that point maybe we start dropping everything in a `bin` dir like the way linux CMake builds are currently set up.

Reply:
That approach would definitely sidestep the RPATH issues I am having with both phonon and steam.

Reply:
I made part of this change -- moved the rpath target properties to a place where they will always apply on macOS (untested, I gotta sleep).  The other part will be to modify `move_lib` so it is always called and writes to the correct place for bundled+nonbundled builds.